include/ozo/type_traits.h

```diff
@@ -506,6 +510,7 @@ OZO_PG_DEFINE_TYPE_AND_ARRAY(float, "float4", FLOAT4OID, FLOAT4ARRAYOID, decltyp
 
 OZO_PG_DEFINE_TYPE_AND_ARRAY(std::string, "text", TEXTOID, TEXTARRAYOID, dynamic_size)
 
+OZO_PG_DEFINE_TYPE(ozo::name_oid, "nameoid", NAMEOID, dynamic_size)
```

Comment:
It seems like this is a "name" type with an array OID = 1003. And it has `NAMEDATALEN` bytes.

    "63-byte type for storing system identifiers"

`NAMEDATALEN` is 64 bytes by default.

https://www.postgresql.org/docs/9.3/static/runtime-config-preset.html

But it seems like it is not good idea to define it as a separate type, since it can be changed while building PostgreSQL.

It is better to use macro for a full type definition:
   
    OZO_PG_DEFINE_TYPE_AND_ARRAY(ozo::pg::name, "name", NAMEOID, 1003, dynamic_size);

Reply:
Done